homeassistant/components/sensibo/strings.json

```diff
@@ -67,6 +69,38 @@
           "off": "Off"
         }
       }
+    },
+    "climate": {
+      "climate_device": {
+        "state_attributes": {
+          "fan_mode": {
+            "state": {
+              "quiet": "Quiet",
+              "low": "Low",
+              "medium": "Medium",
```

Comment:
Should use references as much as possible.

Reply:
Done